### PR TITLE
add get for api/categories and everything required for it

### PR DESCRIPTION
**Pull request by Aelorn13**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const { getCategories } = require("./controllers/categories.js");
+
+const app = express();
+
+app.use(express.json());
```
Comment (Harrison365) [inline app.js]: please remove this and re-add when posting/patching. 😸

Comment (Aelorn13) [reply app.js]: okay!

**Review by Harrison365 — CHANGES_REQUESTED** [review]
Please check the contents of the categories array (for slug and description keys). (Write test)
Please error handle for 404 incorrect paths including testing (Bad path).
Thanks :)

**Comment by Aelorn13** [comment]
oh, my bad I though I need only  remove this part and that's it

**Pull request merged**